src/js/player.js

```diff
@@ -1934,10 +1934,26 @@ class Player extends Component {
       return;
     }
 
-    if (this.paused()) {
-      silencePromise(this.play());
-    } else {
-      this.pause();
+    if (
+      this.options_ === undefined ||
+        this.options_.userActions === undefined ||
+        this.options_.userActions.click === undefined ||
+        this.options_.userActions.click !== false
+    ) {
+
+      if (
+        this.options_ !== undefined &&
+          this.options_.userActions !== undefined &&
+          typeof this.options_.userActions.click === 'function'
```

Comment:
nit: spacing
```suggestion
        this.options_ !== undefined &&
        this.options_.userActions !== undefined &&
        typeof this.options_.userActions.click === 'function'
```

Reply:
Yeah, I rarely edit  javascript and recently switched to doom emacs and don't have the auto indention working right yet.

Fixed the Spacing and reverted the package-lock.json.

I spent a couple hours trying to write the unit tests and it wasn't working out. Like I said, I don't normally write javascript and am not familiar with the test framework. And unfortunately this was already some deep Yak shaving by the Bike Shed at the end of a big detour trying to get contorted aspect ratios and css layout working on my main task that I'm already weeks behind on.

Maybe there is someone lurking who is familiar with the test framework could add that?  Conceptually it should be a trivial variation on the userAction.doubeClick tests. But I was not having any luck with that.

Reply:
I'll see if I can get a few minutes to write a test or two tomorrow.